test/apps/app/src/testApp.ts

```diff
@@ -809,6 +854,87 @@ class TestApp {
       .then(() => {
         document.getElementById('token-msg').innerHTML = 'concurrent test passed';
       });
+    }
+
+  /* eslint-disable max-statements, complexity */
+  async testAuthRequired(): Promise<void> {
```

Comment:
Can this be better implemented as a selenium/webdriver test? An example of a multi-tab test is here: https://github.com/okta/okta-auth-js/blob/master/test/e2e/specs/refreshTokens.js#L101

Having it as a spec will mean that this behavior is protected against regression